slices/xz-utils.yaml

```diff
@@ -0,0 +1,36 @@
+package: xz-utils
+
+essential:
+  - xz-utils_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - liblzma5_libs
+    contents:
+      /usr/bin/lzmainfo:
+      /usr/bin/unxz:
+      /usr/bin/xz:
+      /usr/bin/xzcat:
+  utils:
```

Comment:
This slice should *as a minimum* explain why these extra package dependencies are introduced, and also explain which scripts depend on what

Reply:
Good idea, added some inline comments.